src/bitmessagecli.py

```diff
@@ -206,11 +206,11 @@ def apiData():
             BMConfigParser().get('bitmessagesettings', 'port')
         except:
```

Comment:
if you want, you can for example use
```except BaseException:```
or suppress warning

Reply:
Raised separate PR #1801 for it.